### PR TITLE
Add swagger portfolio manager

### PR DESCRIPTION
**Pull request by Hypermood**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -9,4 +9,14 @@ spring.jpa.database-platform=org.hibernate.dialect.MySQL8Dialect
 stock.api.host=${STOCK_API_HOST:localhost}
 stock.api.port=${STOCK_API_PORT:8081}
 
+
+# Specify the path of the OpenAPI documentation
+springdoc.api-docs.path=/api-docs
```
Comment (VladimirNachev) [inline src/main/resources/application.properties]: Please remove these changes - they are redundant.

File: src/main/java/com/melon/portfoliomanager/configs/OpenApiConfig.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+@Configuration
+public class OpenApiConfig {
+    @Bean
+    public OpenAPI PortfolioManagerOpenAPI() {
+        return new OpenAPI()
+                .info(new Info().title("Portfolio Manager API")
+                        .description("Application for tracking your stocks and getting valuable insights about their value.")
+                        .version("1.0"));
+    }
+}
```
Comment (VladimirNachev) [inline src/main/java/com/melon/portfoliomanager/configs/OpenApiConfig.java]: [nit] Please add new line to the end of file.

File: src/main/resources/application.properties (hunk 1 above)
Comment (Hypermood) [reply src/main/resources/application.properties]: fixed

File: src/main/java/com/melon/portfoliomanager/configs/OpenApiConfig.java (hunk 2 above)
Comment (Hypermood) [reply src/main/java/com/melon/portfoliomanager/configs/OpenApiConfig.java]: fixed

**Pull request merged**